demos/storybook/stories/drawer/with-full-config.tsx

```diff
@@ -218,7 +221,7 @@ export const withFullConfig = (context: DrawerStoryContext): StoryFnReactReturnT
             onClick: (): void => {
                 context.store.set({ selected: schedule });
             },
-            icon: <AirportShuttle />,
+            icon: <AirportShuttle style={getLeftToRightIconDirection()} />,
```

Comment:
This syntax reads cleaner...do you want to also use it for the 3 stories above for consistency?

Reply:
As long as it doesn’t hurt copy-ability of the code